docs/development/module/off-chain-workers.md

```diff
@@ -0,0 +1,383 @@
+---
+title: Off-Chain Workers
+---
+
+You can learn about the high level concept of off-chain workers in our [Conceptual Guide](conceptual/core/off-chain-workers.md).
+
+## Using Off-Chain Workers in Runtime
+
+To use off-chain workers in your runtime, include these following modules
+
+**runtime/src/my_runtime.rs**
```

Comment:
For tutorials I like the idea of providing a specific path to the file being modified. I'm not sure it makes sense here since we don't know the name of the reader's pallet or the filesystem location where it is being developed. It may be in its own rust module rather than right in the runtime.

Reply:
True. But the filename provide the context where the code to be added. I was thinking,

1. skipping the filename altogether - then readers may not know where to add the code. Since some code are added in `lib.rs`, and some in the `runtime.rs`

2. `runtime.rs` (or `pallet.rs`) - this is the best name I can thinking of without the file path

3. `runtime/src/my_runtime.rs` - giving the path, hopefully it gives more clue on where the code to be added

What do you think?

Reply:
Yeah, I agree. Leaving some kind of path is better than no path.

I don't like the name `runtime.rs` for the pallet though because the complete runtime actually lives in `lib.rs`

Reply:
True. Actually I have changed all file paths to `runtime/src/offchain_pallet.rs`.

Refer to this commit: 
https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/pull/379/commits/90316e7925704c376a1012fdcb874a87280bfab2

See if the PR is good for approval. Thanks.